python/hail/dataset.py

```diff
@@ -2039,28 +2039,26 @@ def linreg(self, y, covariates='', root='va.linreg', minac=1, minaf=None):
 
         :param str y: Response sample annotation.
 
-        :param str covariates: Covariant sample annotations, comma separated.
+        :param covariates: list of covariant sample annotations
```

Comment:
These aren't just annotations, these are expressions, right?  You use this language in several places.